qa/tasks/qemu.py

```diff
@@ -37,8 +38,7 @@ def normalize_disks(config):
 
         disks = client_config.get('disks', DEFAULT_NUM_DISKS)
         if not isinstance(disks, list):
-            disks = [{'image_name': '{client}.{num}'.format(client=client,
-                                                            num=i)}
+            disks = [{'image_name': '{typ}.{id_}'.format(typ=typ, id_=id_)}
```

Comment:
Previously, `{client}` was expanded to `client.0` and the resulting image names were `client.0.0`, `client.0.1`, etc.  Replacing `{client}.{num}` with just `{typ}.{id_}` lead to `client.0` being used as a name for all images to be created for that client:
```
2023-12-05T21:51:34.577 INFO:teuthology.run_tasks:Running task qemu...
2023-12-05T21:51:34.610 INFO:tasks.rbd:Creating image client.0 with size 10240
2023-12-05T21:51:34.611 DEBUG:teuthology.orchestra.run.smithi162:> adjust-ulimits ceph-coverage /home/ubuntu/cephtest/archive/coverage rbd --cluster ceph -p rbd create --size 10240 client.0 --image-format 2
2023-12-05T21:51:34.700 INFO:tasks.rbd:Creating image client.0 with size 10240
2023-12-05T21:51:34.700 DEBUG:teuthology.orchestra.run.smithi162:> adjust-ulimits ceph-coverage /home/ubuntu/cephtest/archive/coverage rbd --cluster ceph -p rbd create --size 10240 client.0 --image-format 2
2023-12-05T21:51:34.753 INFO:teuthology.orchestra.run.smithi162.stderr:rbd: create error: (17) File exists
2023-12-05T21:51:34.753 INFO:teuthology.orchestra.run.smithi162.stderr:2023-12-05T21:51:34.753+0000 7f1bcf178e00 -1 librbd: rbd image client.0 already exists
2023-12-05T21:51:34.756 DEBUG:teuthology.orchestra.run:got remote process result: 17
```
Need to resurrect `{num}` here.

Reply:
Done.